README.md

```diff
@@ -19,57 +26,195 @@ pip install nvflare
 ```
 
 ## Quick Start
+The quick start guide means to help the user get FLARE up & running quickly without introducing any advanced concepts. For more details, refer to [Getting Started](https://nvflare.readthedocs.io/en/main/getting_started.html). 
+Since FLARE offers different modes of running the system, we only cover the simplest approaches here. This quick start guide uses the examples/hello-world/hello-numpy-sag as an example. You will find the details in the example's README.md file.
+We also assume you have worked with Python, already set up the virtual env. If you are new to this, please refer [Getting Started](https://nvflare.readthedocs.io/en/main/getting_started.html).
+
+#### Install NVFLARE
+```
+$ python3 -m pip install nvflare
+```
+Clone NVFLARE repo to get examples, switch main branch (latest stable branch)
+```
+$ git clone https://github.com/NVIDIA/NVFlare.git
+$ cd NVFlare
+$ git switch main
+```
+#### Quick start with CLI
+Create a temp directory as workspace and install requirements/dependencies:
+```
+$ mkdir -p /tmp/nvflare
```

Comment:
Do we want to run this in `/tmp/nvflare`?  or create a per-job subdirectory, like `/tmp/nvflare/hello-numpy-run`?  or possibly even run in a local temporary subdir.  I have occasionally had collisions or permission issues when running in /tmp/nvflare - accidentally overwriting output I wanted to save to review, or hitting ownership issues when running with multiple users.